.project

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I missed that earlier, but we usually don't commit IDE specific files in our repositories. See for example https://github.com/SonarSource/sonarlint-eclipse/blob/master/.gitignore

Reply:
I thought they were important. Removed them